src/index.html

```diff
@@ -1,12 +1,336 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
+    <title>Document</title>
+    <link rel="stylesheet" href="styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
+
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="head">
+        <a class="head__icon" href="#">
+          <img src="images/logo.png" alt="icon">
+        </a>
+        <nav class="nav">
```

Comment:
FYI: you could use DEM mixes not to create redundant `div`-wrapper '.head' like this:

```
<body class="page-content">
  <header class="header page-content__header">
    <a class="head__icon" href="#">
      <img src="images/logo.png" alt="icon">
    </a>
    <nav class="nav header__nav">
```

Here `.page-content__header` is element and can have margiins